### ISSUE TITLE
Translate Program to various languages

### ISSUE THREAD

**Issue by albar965** [albar965/littlenavmap#135]
Let me know if you're volunteering for a translation to German or any other language.

**Comment by merspieler** [comment]
I could translate to german

**Comment by albar965** [comment]
Thank you for the offer. Send me an email. Mail address is on https://albar965.github.io/
Alex

**Comment by Cele91** [comment]
I can translate in Italian :)

**Comment by albar965** [comment]
Hi Cele91, thank you for the offer. I pointed Valerio who is already working on the Italian translation to this thread. Maybe you can share the work.
Alex

**Comment by efa** [comment]
hi,
you can dowload the current IT translation file from:
https://github.com/efa/littlenavmap/tree/feature/translation
it contain 350 translated of 2350 strings, so 2k are missing.

We can partition the remaining in two 1k strings, and work in parallel to speed up work.

Here a proposal:
AirportSearch	41
DatabaseManager	46
OptionsDialog	54
RouteController	87
MapWidget	88
Options	230
MapHtmlInfoBuilder	380
to me, and:

QObject	387
MainWindow	686
to you.

Let me know if it is OK for you?

**Comment by efa** [comment]
Alex maybe it is simplest if I ask for a pull request of current status in my fork, so Cele91 can fork the main repository?

**Comment by albar965** [comment]
@efa: As mentioned in the email: Yes, do that pull request.
@Cele91: Hope that is ok with you.
Alex

**Comment by Cele91** [comment]
@efa : I can try. It's the first time I open Qt Linguist and so I think I can do it. Can I have your contact details for any questions?  My mail - cele9191@gmail.com

**Comment by efa** [comment]
replyed via email

**Comment by efa** [comment]
@albar965 pull done

**Comment by efa** [comment]
@Cele91
Let me know if it is OK for you to work on strings:
QObject 387
MainWindow 686
?

**Comment by Cele91** [comment]
@efa Affirmative, but slowly for now. Did you write to me? I have not received any email

**Comment by efa** [comment]
yes, I wrote to you. Check in the Spam folder using IMAP or webmail, sometimes gmail consider spam some mail from my provider

**Comment by efa** [comment]
I just updated my fork translation branch to 2.0.1b:
https://github.com/efa/littlenavmap/tree/feature/translation
it contain 644 translated of 2829 strings, so 2.2k are missing.

AirportSearch 8/43
DatabaseManager 1/47
RouteExport 0/51
UserdataController 3/52
OptionsDialog 3/54
MapVisible 2/62
RouteController 8/97
Options 2/294
QObject 37/394
MapHtmlInfoBuilder 30/428
MainWindow 22/786

**Comment by FantomasicPlay** [comment]
I could translate to russian

**Comment by albar965** [comment]
> I could translate to russian

Perfect. Very welcome. Nobody volunteered yet for Russian so there won't be any conflicts.

See here for more information:
[Translating the Program](https://github.com/albar965/littlenavmap/wiki/Translating-the-Program)

Feel free to ask in the [discussions](https://github.com/albar965/littlenavmap/discussions) or send me an email if something is not clear.

**Comment by tonghuix** [comment]
I am back, translating Chinese.

**Comment by albar965** [comment]
> I am back, translating Chinese.

Great. Welcome back!

**Comment by Cat-cop** [comment]
hi! i could try translate to Russia

**Comment by albar965** [comment]
> hi! i could try translate to Russia

That's great! Have a look at the instructions here [Translating the Program](https://github.com/albar965/littlenavmap/wiki/Translating-the-Program) and do not hesitate to ask (email or here, doesn't matter) if something is not clear.


Best,
Alex

**Comment by Cat-cop** [comment]
# @albar965 where i can find an EN version for QT i just can't find

**Comment by albar965** [comment]
> # [@albar965](https://github.com/albar965) where i can find an EN version for QT i just can't find

There is no EN translation file. English is built into the code and shown in the Qt Linguist translation tool.

<img width="1324" height="826" alt="Image" src="https://github.com/user-attachments/assets/471f561e-cdd1-4e16-be8b-1e5e663afd22" />
